ports/fnafnes/FNAF NES.sh

```diff
@@ -0,0 +1,56 @@
+XDG_DATA_HOME=${XDG_DATA_HOME:-$HOME/.local/share}
+
+if [ -d "/opt/system/Tools/PortMaster/" ]; then
+  controlfolder="/opt/system/Tools/PortMaster"
+elif [ -d "/opt/tools/PortMaster/" ]; then
+  controlfolder="/opt/tools/PortMaster"
+elif [ -d "$XDG_DATA_HOME/PortMaster/" ]; then
+  controlfolder="$XDG_DATA_HOME/PortMaster"
+else
+  controlfolder="/roms/ports/PortMaster"
+fi
+
+source $controlfolder/control.txt
+source $controlfolder/device_info.txt
+export PORT_32BIT="Y"
+
+get_controls
+[ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"
+
+$ESUDO chmod 666 /dev/tty0
+
+GAMEDIR="/$directory/ports/fnafnes"
+
+export LD_LIBRARY_PATH="/usr/lib32:$GAMEDIR/libs"
+export GMLOADER_DEPTH_DISABLE=1
+export GMLOADER_SAVEDIR="$GAMEDIR/gamedata/"
+export GMLOADER_PLATFORM="os_linux"
+
+# We log the execution of the script into log.txt
+exec > >(tee "$GAMEDIR/log.txt") 2>&1
+
+cd $GAMEDIR
+
+if [ -f "${controlfolder}/libgl_${CFWNAME}.txt" ]; then 
+  source "${controlfolder}/libgl_${CFW_NAME}.txt"
+else
+  source "${controlfolder}/libgl_default.txt"
+fi
+
+# Check for file existence before trying to manipulate them:
+[ -f "./gamedata/data.win" ] && mv gamedata/data.win gamedata/game.droid
+[ -f "./gamedata/game.win" ] && mv gamedata/game.win gamedata/game.droid
+[ -f "./gamedata/game.unx" ] && mv gamedata/game.unx gamedata/game.droid
+
+# Make sure uinput is accessible so we can make use of the gptokeyb controls
+$ESUDO chmod 666 /dev/uinput
+
+$GPTOKEYB "gmloader" -c ./fnafnes.gptk &
+
+$ESUDO chmod +x "$GAMEDIR/gmloader"
+
+./gmloader fnafnes.apk
+
+$ESUDO kill -9 $(pidof gptokeyb)
+$ESUDO systemctl restart oga_events &
+printf "\033c" > /dev/tty0
```

Comment:
The launch script template has been updated. Now we just call `pm_finish` See [packaging](https://portmaster.games/packaging.html#the-launchscript-sh) :)
```suggestion
pm_finish
```